exercises/beer-song/inputdata.txt

```diff
@@ -0,0 +1,7 @@
+99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n
+44 bottles of beer on the wall, 44 bottles of beer.\nTake one down and pass it around, 43 bottles of beer on the wall.\n
```

Comment:
Perhaps I was lazy.  I just pasted in the literal strings from the canonical data.  Delphi (pascal) has no idea what a `\n` character is.  In pascal I would have to write my own routine to scan for occurances of `\n` and replace it with `#13#10` (crlf).  I thought I would take the easy route..